frontend/src/scenes/PreflightCheck/logic.ts

```diff
@@ -94,7 +95,12 @@ export const preflightLogic = kea<preflightLogicType<PreflightStatus, PreflightM
     }),
     events: ({ actions }) => ({
         afterMount: () => {
-            actions.loadPreflight()
+            const preflight = getAppContext()?.preflight
+            if (preflight) {
+                actions.loadPreflightSuccess(preflight)
```

Comment:
Minor thing, `loadPreflight` actually calls `registerInstrumentationProps` to set up some super props on PostHog events, but we're bypassing the action here, probably we just need to move it to the listener.

Reply:
Oh, how did I miss that (I know, it was 1am...)...

Fixed now. And in general, it's best to do work like this in a success listener.